### PR TITLE
Added pair_list, scoped_key_value_pairs, and key_value_pair nodes.

### PR DESCRIPTION
**Pull request by armcburney**

### CODE REVIEW

File: cpp/src/nodes/scoped_key_value_pairs.hpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#ifndef SCOPEDKEYVALUEPAIRS_H
+#define SCOPEDKEYVALUEPAIRS_H
+
+#include "node.hpp"
+
+/**
+ * ScopedKey value pairs, scoped to a particular base_keyword in 'pair_list' rule
```
Comment (armcburney) [inline cpp/src/nodes/scoped_key_value_pairs.hpp]: oops, change to `Scoped key value pairs, ...`

File: cpp/src/nodes/node_list.cpp [hunk 2]
```diff
@@ -1,8 +1,8 @@
-#include "root.hpp"
+#include "node_list.hpp"
 
-Root::Root(NodePtrs elements) : elements(elements) {}
+NodeList::NodeList(NodePtrs elements) : elements(elements) {}
 
-std::string Root::to_html(Json &context) {
+std::string NodeList::to_html(Json &context) {
```
Comment (armcburney) [inline cpp/src/nodes/node_list.cpp:5]: 🦆

File: cpp/src/nodes/pair_list.cpp [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#include "pair_list.hpp"
+
+PairList::PairList(NodePtrs key_value_pairs): key_value_pairs(key_value_pairs) {}
```
Comment (armcburney) [inline cpp/src/nodes/pair_list.cpp]: Add a space in between the initializer list.

**Review by davepagurek — APPROVED** [review]
💦

**Pull request merged**